README.md

```diff
@@ -1 +1,185 @@
-# portfolio-austin
\ No newline at end of file
+# portfolio
+<a name="readme-top"></a>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Portfolio <a name="about-project"></a>
+
+>  
+
+**Portfolio** is project that shows some of the moile first designs that I have made at Microverse.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://html.com/">HTML</a></li>
+     <li><a href="http://css.com/">CSS</a></li>
+  </ul>
+</details>
+
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+
+- **Header**
+- **headline section**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+Code editor like Visual studio
+windows or Linux operating systems
+2GB RAM
+20G HDD
+
+### Setup
+
+Clone this repository to your desired folder using the following commands:
+
+
+  cd your-folder
+  git clone https://github.com/ausiemass/portfolio
+
+
+### Install
+
+Install this project with:
+
+
+  cd my-project
+  gem install
+
+
+
+### Usage
+
+To run the project, execute the following command:
+
+
+Shift + ctrl + R
+
+
+### Run tests
+
+To run tests, run the following command:
+
+
+Shift + ctrl + T
+
+
+### Deployment
+
+You can deploy this project using:
+<ul>    
+<li>Windows</li>
+<li>Linux</li>
+</ul>
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+
+
+👤 **Austin Masamhiri**
+
+- GitHub: [@ausiemass](https://github.com/)
+- Twitter: [@ausiemass](https://twitter.com/home)
+- LinkedIn: [ausiemass](https://www.linkedin.com/feed/)
```

Comment:
- [ ] [Optional] 👍 Great job on including an author section in your project! It's important to provide information about the project's author. Regarding the links you've added for GitHub, Twitter, and LinkedIn, I have a suggestion for improvement:

Instead of linking to the home pages of these platforms, it is recommended to use your personal profile links. By linking to your profile, visitors can directly access your account and learn more about your work and contributions.

To implement this improvement, please 🙏 update the links in the author section to point to your specific profile pages on GitHub, Twitter, and LinkedIn. Keep up the good work and make sure to provide visitors with easy access to your personal profiles! 👍🔗